.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+CHANGELOG: `no-impact` / `Added` / `Changed` / `Deprecated` / `Removed` / `Fixed` / `Security`
+
+Describe how these changes affect the customer in one line. This is the thing you are going to add to a changelog.txt file.
+
+> CHANGELOG entries will be pulled directly into the release changelog, so use
+customer-facing language. Start with one of the active verbs above,
+write a full sentence, and end with a period. If these changes do not
+impact the customer, then this line should read CHANGELOG: no-impact.
```

Comment:
This doesn't seem to fit the way we do changelogs. Is it from some other template?

Reply:
This is lifted from [here](https://eng-handbook.hashicorp.services/git-and-github/#templates), which AFAIK, is a terraform checklist.

Are you saying that the contents themselves are not applicable to us? I think it could be valuable to follow that pattern for our changelogs too 🤔

Reply:
I'm not sure how the no-impact option fits in with the our automated changelog generation tool. It might be worth including the keywords that it looks at for convenience. I believe the pattern is ```release-note: 
followed by one of breaking-change, bug, deprecation, enhancement, feature, improvement, note, security
We historically have been using lowercase, and I'd argue for keeping it that way for consistiency.

Reply:
I don't think this belongs in the PR description, as it's already enforced by a GitHub action and (as others have said) doesn't match the way we actually manage our changelog.

Reply:
Thanks everyone for the feedback -- per that, I deleted this section.